### ISSUE TITLE
Facilitating the MarryTTS default voices installation process

### ISSUE THREAD

**Issue by kosmasK** [marytts/marytts#617]
Hello MaryTTS community! 
I have downloaded from the releases the source code of marytts v5.2 and build it successfully. However,  I am trying to find a way to import the available default voices without doing it manually by downloading and building each voice separately. Is it possible to do something similar to the marytts-installer 's command :
> marytts 'install:*' 

Is it possible to transfer the .jar files of the voices from the marytts-installer to the marrytts-5.2 file system? Also I am not able to find any voices folder in my local marytts folder. In general I am working through terminal on a VM server and I would like to know if I can also build my own voices without the use of GUIs.
I would appreciate your help guys! Thank you in advance!

**Comment by seblemaguer** [comment]
Hello,

you should have a look to this repository : https://github.com/marytts/marytts-installer

**Comment by seblemaguer** [comment]
ok sorry, I misread. One solution, is to : 
1. download all the voices using the installer,
2. add all the jar in the classpath when running the marytts server

you do not need to move anything.

**Comment by kosmasK** [comment]
@seblemaguer Thank you for your quick response to my question!
I have downloaded all the voices with marytts-installer. The jar files are placed at the 
`/marytts-installer/installed/` folder. 
However I am not using the server binary of the marytts-installer distribution, but the https://github.com/marytts/marytts/tree/v5.2 distribution and the server is under
`marytts/marytts-5.2/target/marytts-5.2/bin/` folder. 
Thus I want the bin of this distribution to be able to locate the voices jars. 
The classpath is specified as `CLASSPATH=$APP_HOME/gradle/wrapper/gradle-wrapper.jar` only at the marytts bin of marytts-installer.
And still I am wondering if I can do the installations as a batch without following the given instructions for manually downloading and building each voice individually as described  below (it will take a lot of time to do so) :

> To assemble, test, and package the voice for another MaryTTS installation, do
`$ ./gradlew build`
The packaged files will be placed under build/distributions. Copy the zip file and xml descriptor into your MaryTTS installation's download directory, then run the marytts-component-installer to install the voice.

Thanks again for your help!

**Comment by psibre** [comment]
You don't have to use the legacy InstallerGUI. You can just download and unzip the release assets from the voices that are available. E.g., to make the dfki-poppy voice available (without "installing" it) on your headless remote server, you can just run something like
```sh
$ cd path/to/the/marytts/installation
$ wget https://github.com/marytts/voice-dfki-poppy/releases/download/v5.2/voice-dfki-poppy-5.2.zip
$ unzip voice-dfki-poppy-5.2.zip
$ mv voice-dfki-poppy-5.2.zip download # for good measure
$ bin/marytts-server
```
You'll notice that the data under `lib/` is *almost* the same as that under the `marytts-installer`'s `installed/`. So you can also copy those over, just make sure the jars are put *under* (not beside) the `lib/`.

Not that I understand exactly what you intend to do. In fact, you mention that you want to build your own voices... That would be a separate issue.

**Comment by kosmasK** [comment]
@psibre Thanks for your answer! Even thought you did not understand me totally, your answer helped me to grasp how the voice libraries work.
I know that building a new voice is a completely different process and currently I am trying to collect and edit some voice data in order to follow the tutorials of the wiki. I appreciate your fast responses and your support! 
Cheers!

**Comment by seblemaguer** [comment]
what do you mean by ```edit some voice data ``` ?

**Comment by kosmasK** [comment]
@seblemaguer 
Currently I am trying to build a dataset for a new voice. While preparing my data I noticed that the already existing mary voice datasets have utterances which generally range from 2 sec to 5 sec. So, I treated this as a rule and took care so that my utterances' durations are time-limited accordingly. Also I think that it requires to have silences smaller than 200ms in each utterance. 

However, I was wondering if it plays role the structure of the utterance in order to achieve the correct intonation. For example, should my utterances end with a full stop? Can they include commas? Should they be grammatically correct, with a meaning or they can be just simple words?

I know that the wavs need to be aligned with the text and the phonemes but I am not sure what are the exact rules of the utterances in order to meet the requirements of a proper training process. I would appreciate some enlightenment and guidance on those issues!

Peace!
